src/mount/linux.rs

```diff
@@ -161,3 +163,41 @@ pub fn umount2<P: ?Sized + NixPath>(target: &P, flags: MntFlags) -> Result<()> {
 
     Errno::result(res).map(drop)
 }
+
+libc_bitflags!(
+    /// Flags for [`open_tree()`].
+    pub struct OpenTreeFlags: c_uint {
+        /// Directly query the mount attached to the file descriptor.
+        AT_EMPTY_PATH as c_uint;
+        /// Do not trigger an automount target.
+        AT_NO_AUTOMOUNT as c_uint;
+        /// When cloning, also clone nested mount subtrees.
+        AT_RECURSIVE as c_uint;
+        /// If target is a symbolic link, do not dereference it.
+        AT_SYMLINK_NOFOLLOW as c_uint;
+        /// Clone the mount object.
+        OPEN_TREE_CLONE as c_uint;
```

Comment:
Is this cast necessary, it seems that this constant is of type `c_uint` on both Linux and Android:

```sh
$ rg "OPEN_TREE_CLONE"
src/unix/linux_like/android/mod.rs
2174:pub const OPEN_TREE_CLONE: ::c_uint = 0x01;

src/unix/linux_like/linux/mod.rs
3328:pub const OPEN_TREE_CLONE: ::c_uint = 0x01;

libc-test/semver/linux.txt
1728:OPEN_TREE_CLONE

libc-test/semver/android.txt
1754:OPEN_TREE_CLONE
```